tests/integration/devfile/cmd_devfile_create_test.go

```diff
@@ -99,6 +101,33 @@ var _ = Describe("odo devfile create command tests", func() {
 			expectedFiles := []string{"package.json", "package-lock.json", "README.md", devfile}
 			Expect(helper.VerifyFilesExist(newContext, expectedFiles)).To(Equal(true))
 		})
+
+		It("should successfully create the devfile component and show json output for working cluster", func() {
+			helper.CopyExampleDevFile(filepath.Join("source", "devfiles", "nodejs", "devfile.yaml"), filepath.Join(devfilePath))
+			output := helper.CmdShouldPass("odo", "create", "nodejs", "--context", newContext, "-o", "json")
+			values := gjson.GetMany(output, "kind", "metadata.name", "status.state")
+			Expect(helper.GjsonMatcher(values, []string{"Component", "nodejs", "Not Pushed"})).To(Equal(true))
+		})
+
+		It("should successfully create and push the devfile component and show json output for working cluster", func() {
+			helper.CopyExampleDevFile(filepath.Join("source", "devfiles", "nodejs", "devfile.yaml"), filepath.Join(devfilePath))
+			output := helper.CmdShouldPass("odo", "create", "nodejs", "--starter", "--context", newContext, "-o", "json", "--now")
+			expectedFiles := []string{"package.json", "package-lock.json", "README.md", devfile}
+			Expect(helper.VerifyFilesExist(newContext, expectedFiles)).To(Equal(true))
+			Expect(output).To(ContainSubstring("Pushed"))
+			Expect(output).To(ContainSubstring("nodejs"))
+			Expect(output).To(ContainSubstring("Component"))
```

Comment:
We can use `helper.MatchAllInOutput()` instead

Reply:
ack

Reply:
done